iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/ModificationReports.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.modification;
+
+import com.powsybl.commons.reporter.Report;
+import com.powsybl.commons.reporter.Reporter;
+import com.powsybl.commons.reporter.TypedValue;
+import com.powsybl.iidm.network.BusbarSection;
+import com.powsybl.iidm.network.IdentifiableType;
+import com.powsybl.iidm.network.Injection;
+
+/**
+ * @author Coline Piloquet <coline.piloquet at rte-france.com>
+ */
+public final class ModificationReports {
```

Comment:
It seems to be really specific to topology modifications, maybe it should be in topology(/util?) and be package private rather than public